validator/main.py

```diff
@@ -9,14 +10,15 @@
 def main():
     "Main function. Handles delegation to other functions."
 
+    logging.basicConfig()
```

Comment:
Yeah, otherwise stack traces don't get reported, and we get a `No handlers could be found for logger "amo.validator"` message when running from the command line. (The current release already emits  `No handlers could be found for logger "root"` messages from time to time for the same reason.)